src/frameworks/drupal7/simplesaml.md

```diff
@@ -96,6 +101,22 @@ if (isset($_ENV['PLATFORM_PROJECT_ENTROPY'])) {
 }
 ```
 
+## Generate SSL certs (optional)
+
```

Comment:
What's SP?

Reply:
> generate the cert

Let's use the full word?

Reply:
"is should"?

Reply:
"Whatever the file is" / "should be placed in".  It works grammatically, although I'm open to alternatives.

Reply:
Resolved